app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id].to_i)
+    @user = User.includes(posts: %i[comments author]).find(params[:user_id].to_i)
```

Comment:
- [ ] the to fix the N+1 issue in the post controller the index method should look something like below:
```
@posts = @user.posts.includes(:comments)
```

Reply:
Thankyou michael 😊